cmd/utils/flags.go

```diff
@@ -1885,9 +1885,6 @@ func RegisterEthService(stack *node.Node, cfg *ethconfig.Config) (ethapi.Backend
 			Fatalf("Failed to create the LES server: %v", err)
 		}
 	}
-	if err := ethcatalyst.Register(stack, backend); err != nil {
```

Comment:
Why to remove it?

Reply:
I move it to [here](https://github.com/ethereum/go-ethereum/pull/27327/files#diff-efb34882aeb5938c3c2527733e914b31697acefcdccc3b3c42467fda64efc362R196-R205) .

The `SimulatedBeacon` uses its own instance of `ConsensusAPI`.  I avoid instantiating it twice because it opens the auth RPC API port by default and causes some useless log warnings (https://github.com/ethereum/go-ethereum/pull/27327#issuecomment-1594507869).